File: explorer/src/main/java/com/duongame/helper/UnitHelper.java
package com.duongame.helper;

import android.content.res.Resources;

/**
 * Created by namjungsoo on 2016-11-19.
 */

public class UnitHelper {
    public static int dpToPx(int dp) {
        return (int) (dp * Resources.getSystem().getDisplayMetrics().density);
    }

    public static int pxToDp(int px) {
        return (int) (px / Resources.getSystem().getDisplayMetrics().density);
    }
}

File: Explorer/src/main/java/com/duongame/helper/UnitHelper.kt
package com.duongame.helper

import android.content.res.Resources

/**
 * Created by namjungsoo on 2016-11-19.
 */
object UnitHelper {
    @JvmStatic
    fun dpToPx(dp: Int): Int {
        return (dp * Resources.getSystem().displayMetrics.density).toInt()
    }

    fun pxToDp(px: Int): Int {
        return (px / Resources.getSystem().displayMetrics.density).toInt()
    }
}